blazor/dropdown-list/grouping.md

```diff
@@ -7,53 +7,36 @@ control: DropDown List
-    <DropDownListFieldSettings GroupBy="Category" Value="Vegetable"></DropDownListFieldSettings>
-</SfDropDownList>
-
-@code {
-
-    public IEnumerable<Vegetables> LocalData { get; set; } = new Vegetables().VegetablesList();
-
-    public class Vegetables
-    {
-        public string Vegetable { get; set; }
-        public string Category { get; set; }
-        public string ID { get; set; }
-
-        public List<Vegetables> VegetablesList()
-        {
-            List<Vegetables> Veg = new List<Vegetables>();
-            Veg.Add(new Vegetables { Vegetable = "Cabbage", Category = "Leafy and Salad", ID = "item1" });
-            Veg.Add(new Vegetables { Vegetable = "Chickpea", Category = "Beans", ID = "item2" });
-            Veg.Add(new Vegetables { Vegetable = "Garlic", Category = "Bulb and Stem", ID = "item3" });
-            Veg.Add(new Vegetables { Vegetable = "Green bean", Category = "Beans", ID = "item4" });
-            Veg.Add(new Vegetables { Vegetable = "Horse gram", Category = "Beans", ID = "item5" });
-            Veg.Add(new Vegetables { Vegetable = "Nopal", Category = "Bulb and Stem", ID = "item6" });
-            Veg.Add(new Vegetables { Vegetable = "Onion", Category = "Bulb and Stem", ID = "item7" });
-            Veg.Add(new Vegetables { Vegetable = "Pumpkins", Category = "Leafy and Salad", ID = "item8" });
-            Veg.Add(new Vegetables { Vegetable = "Spinach", Category = "Leafy and Salad", ID = "item9" });
-            Veg.Add(new Vegetables { Vegetable = "Wheat grass", Category = "Leafy and Salad", ID = "item10" });
-            Veg.Add(new Vegetables { Vegetable = "Yarrow", Category = "Leafy and Salad", ID = "item11" });
-            return Veg;
-        }
-    }
-}
-```
-
-The output will be as follows.
-
-![Grouping in Blazor DropdownList](./images/blazor-dropdownlist-grouping.png)
\ No newline at end of file
+In the following sample, vegetables are grouped according on its category using `DropDownListFieldSettings.GroupBy` field.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/grouping/grouping.razor %}
+
+{% endhighlight %}
+
+![Grouping in Blazor DropdownList](./images/grouping/blazor-dropdownlist-grouping.png)
+
+## Fixed group header
+
+You can classify the items based on the [DropDownListFieldSettings.GroupBy](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.DropDownListFieldSettings.html#Syncfusion_Blazor_DropDowns_DropDownListFieldSettings_GroupBy) field mapped to the control. The headers of group items will fixed at the top till its belonging items ends up on scrolling.
+
+## Group header template
+
+The group header title under which appropriate sub-items are categorized can also be customize with the help of [GroupTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_GroupTemplate) property. This template is common for both inline and floating group header template.
```

Comment:
The group header title under which appropriate sub-items are categorized is customize with the help of the [GroupTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_GroupTemplate) property. This template is common for both inline and floating group header template.